extra/prometheus/prometheus.yml

```diff
@@ -23,14 +23,15 @@ scrape_configs:
     static_configs:
     - targets: ["prometheus-server.fqdn.com:9090"]
 
-# ################################ #
-# Pure Storage FlashArray Number 1 #
-# ################################ #
+# ################################## #
+# Pure Storage FlashArray Number 1   #
+# Running Purity//FA 6.7.0 or higher #
+# ################################## #
 # Create a single scrape job for each selected metric endpoint per FlashArray.
 # It is NOT recommended to scrape the root /metrics endpoint as it can be an expensive query.
 # Scrape job for Pure Storage FlashArray 1 scraping /metrics/array
   - job_name: 'purefa_array_arrayname01'
-    metrics_path: /metrics/array
+    metrics_path: /metrics/array?namespace=purefa
```

Comment:
This should be added as a parameter and replace “endpoint” which is also not required anymore.